tooling/strace_analysis.sh

```diff
@@ -169,15 +169,24 @@ filterStraceFiles() {
 processFiles() {
     echo "Processing found files to determine 'Package' versions... (this will take a few minutes)"
 
+    # Determine OS package query command
+    os_type=centos
+    package_query="rpm -qf"
```

Comment:
Could we move this to an `else` clause in the if block below just to make the logic a bit more explicit now that we have multiple options?
Defaulting to `rpm` makes me a little nervous, but since I haven't got an easy check for that situation (Unless we do something like 'if which rpm` in the else clause just to verify that rpm is there even if it isn't the default) and it is likely to only be edge cases such as gentoo that we miss I'm ok with letting that go :-)

Reply:
yep done

Reply:
I'm going to be extra picky here and say that the rpm check should explicitly be done after all the others. There are valid reasons why someone may have `rpm` installed on a Debian/Ubuntu box (rpm is even in the default Ubuntu repositories) so the check for `rpm` should be the last one.

Reply:
(Oh and just in case you're wondering, installing `rpm` on Debian/Ubuntu doesn't give it access to the dpkg database, so you wouldn't be able to use that to work around the need for all the new logic in this PR ;-) )

Reply:
yep makes sense, didn't realize rpm can be on others...!